### PR TITLE
header draft

### PR DESCRIPTION
**Pull request by kq333**
[DEMO LINK](https://kq333.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://kq333.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by michalcywka — CHANGES_REQUESTED** [review]
Links in description are missing

File: src/style.css [hunk 1]
```diff
@@ -1 +1,95 @@
...
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+nav {
```
Comment (michalcywka) [inline src/style.css:23]: Don't style `nav` tags, use classes

**Review by michalcywka — CHANGES_REQUESTED** [review]
Links provided in the description are not working, did you push to github pages?

**Comment by kq333** [comment]
> Links in description are missing

seems ok now.

**Review by michalcywka — CHANGES_REQUESTED** [review]
Please read the task description carefully, not all links change to blue colour on hover.

**Review by michalcywka — CHANGES_REQUESTED** [review]
The first and the second test are failing because whole page is too wide. I have to scroll to see the last item in the navigation menu.

**Review by michalcywka — CHANGES_REQUESTED** [review]
We don't add new lines after class names in classes :)

File: src/style.css [hunk 2]
```diff
@@ -1 +1,86 @@
-/* Styles go here */
+/* Styles go here */
+
+body {
+  margin: 0;
+
```
Comment (michalcywka) [inline src/style.css:5]: There should be no new lines at the end of class definitions

**Review by michalcywka — CHANGES_REQUESTED** [review]

**Review by michalcywka — APPROVED** [review]

**Pull request closed without merging**